tests/e2e/e2e_setup_test.go

```diff
@@ -84,12 +84,19 @@ func (s *IntegrationTestSuite) SetupSuite() {
 		s.T().Log("skipping state sync testing")
 	}
 
+	isDebugLogEnabled := false
+	if str := os.Getenv("OSMOSIS_E2E_DEBUG_LOG"); len(str) > 0 {
+		isDebugLogEnabled, err = strconv.ParseBool(str)
+		s.Require().NoError(err)
+		s.T().Log("debug logging is enabled. container logs from running cli commands will be printed to stdout")
```

Comment:
Confused about this log. I know its just in the test file but wouldn't this be triggered on both True and False?

Reply:
good point, will fix